Formula/llvm.rb

```diff
@@ -120,11 +126,32 @@ def install
     end
 
     on_linux do
+      args << "-DCMAKE_CXX_FLAGS=-fpermissive"
+      args << "-DCMAKE_C_FLAGS=-fpermissive"
```

Comment:
If you set this note that CMake will ignore the `CFLAGS` and `CXXFLAGS` environment variables.

Reply:
Ah, so it's probably better to use `ENV.append` for this instead.